src/rdkafka_conf.h

```diff
@@ -32,6 +32,9 @@
 #include "rdlist.h"
 #include "rdkafka_cert.h"
 
+#if WITH_SSL
+#include <openssl/engine.h>
```

Comment:
Doesn't this need to be version-guarded to 1.1.0 too?

Reply:
doesn't hurt either way since actual variable definition and usage are conditioned.
also engine.h was introduced way back but some methods which i am using were introduced in 1.1.0
but will add the condition for consistency

Reply:
Oh, if openssl/engine.h is available in 1.0.2 (our oldest supported version) it is fine to leave this as-is.